ethereum-consensus/src/electra/epoch_processing.rs

```diff
@@ -0,0 +1,189 @@
+use crate::electra::{
+    compute_activation_exit_epoch, get_activation_exit_churn_limit, get_active_balance,
+    get_current_epoch, increase_balance, initiate_validator_exit, is_active_validator,
+    is_eligible_for_activation, is_eligible_for_activation_queue, switch_to_compounding_validator,
+    BeaconState, Context, Error,
+};
+use ssz_rs::List;
+
+use super::decrease_balance;
```

Comment:
generally prefer to import things in a fully-qualified manner from the crate root

```suggestion
use crate::electra::helpers::decrease_balance;
```